front/components/VideoPlayer/VideoPlayer.spec.tsx

```diff
@@ -44,4 +59,30 @@ describe('VideoPlayer', () => {
     instance.componentWillUnmount();
     expect(instance.state.player!.destroy).toHaveBeenCalled();
   });
+
+  describe('in an EME enabled environment', () => {
```

Comment:
The first E in EME is Encrypted. Is it relevant?

Reply:
I think it's for DRM.